network/Cargo.toml

```diff
@@ -62,6 +62,9 @@ version = "0.2"
 [dependencies.derivative]
 version = "2"
 
+[dependencies.eigenvalues]
```

Comment:
```suggestion
[dev-dependencies.eigenvalues]
```

Reply:
Good catch, this can actually be removed entirely 👍